src/hooks/useDebounce2.ts

```diff
@@ -0,0 +1,15 @@
+import {useEffect, useState} from 'react';
+
+export const useDebounce2 = (value: any, delay = 500) => {
```

Comment:
`useDebounce2` ?

Reply:
Wrong file version from this PR. Deleted file.